src/Umbraco.Web/PropertyEditors/ValueConverters/ContentPickerPropertyConverter.cs

```diff
@@ -75,10 +77,10 @@ public override object ConvertDataToSource(PublishedPropertyType propertyType, o
         {
             var attemptConvertInt = source.TryConvertTo<int>();
```

Comment:
I found it like this.. to be fair I'm not entirely sure what this is for, but if I don't do this conversion then `ConvertSourceToObject` does not seem to get the correct data. I'll check what this method is used for.